templates/admin/user/edit.tmpl

```diff
@@ -186,7 +186,7 @@
 
 				<div class="field">
 					<button class="ui green button">{{$.locale.Tr "settings.update_avatar"}}</button>
-					<a class="ui red button delete-post" data-request-url="{{.Link}}/avatar/delete" data-done-url="{{.Link}}">{{$.locale.Tr "settings.delete_current_avatar"}}</a>{{/* TODO: Convert links without href to buttons for a11y */}}
+					<button class="ui red button link-action" data-url="{{.Link}}/avatar/delete" data-redirect="{{.Link}}">{{$.locale.Tr "settings.delete_current_avatar"}}</button>
```

Comment:
Can we also add a `data-method` that defaults to `POST`? It would be useful, and in this case even better if the request would be `DELETE {{.Link}}/avatar`.

Reply:
No such requirement in code base, so no need to over design at the moment.

Reply:
Guess it can be done later. `data-body` support would also be nice for JSON body for example.